cms/static/js/personalise.js

```diff
@@ -165,6 +166,13 @@ if (personaliseDiv) {
     });
   }
 
+  function clickListeners () {
+    var pyrSearch = select('#pyr-search');
+    pyrSearch.addEventListener('click', function (e) {
+      app.ports.clickScroll.send(scrollToElement(e, '#results'));
```

Comment:
This is another part I know must be wrong. Calling a function here doesn't send a value, but instead performs an action in JS. If it is staying as a port and listening to JS, then I need to send something to elm.